src/overlays/actors/ovl_En_Ossan/z_en_ossan.c

```diff
@@ -20,227 +129,2379 @@ const ActorInit En_Ossan_InitVars = {
+        func_80AC37A8(globalCtx, this);
+        return 1;
+    } else {
+        return 0;
+    }
+}
+
+s32 func_80AC38C8(EnOssan* this, GlobalContext* globalCtx, Input* cont1) {
+    if (CHECK_PAD(cont1->press, B_BUTTON)) {
+        this->unk_1FC = this->unk_1FE;
+        func_8010B720(globalCtx, this->unk_200[this->unk_252]->actor.textId);
+        return 1;
+    } else {
+        return 0;
+    }
+}
+
+void func_80AC3928(GlobalContext* globalCtx, EnOssan* this, u8 arg2) {
+    YREG(31) = 1;
+    this->unk_1F0 = 0;
+    this->unk_1EE = this->unk_1F0;
+    Interface_SetDoAction(globalCtx, 0x10);
+    func_80AC3350(this, globalCtx, 0.0f);
+    if (!arg2) {
+        this->unk_1FC = 1;
+    } else {
+        func_80AC39AC(globalCtx, this);
+    }
+}
+
+void func_80AC39AC(GlobalContext* globalCtx, EnOssan* this) {
+    this->unk_1FC = 2;
+    if (this->actor.params == 0xA) {
+        if ((gSaveContext.itemGetInf[3] & 0x100) && (gSaveContext.itemGetInf[3] & 0x200) &&
+            (gSaveContext.itemGetInf[3] & 0x400) && (gSaveContext.itemGetInf[3] & 0x800)) {
+            // Choose a mask with left or right
+            // on [Control Stick].
+            // Ask about Mask of Truth
+            // Don't borrow a mask 
+            func_8010B720(globalCtx, 0x70AD);
+        } else {
+            // Select a mask with left or right
+            // on [Control Stick].
+            // Talk to the shop owner.
+            // Don't borrow 
+            func_8010B720(globalCtx, 0x70A2);
+        }
+    } else {
+        // Shop around by moving the
+        // [Control Stick] left or right.
+        // Talk to the owner
+        // Quit 
+        func_8010B720(globalCtx, 0x83);
+    }
+
+    Interface_SetDoAction(globalCtx, 6);
+    this->unk_2C0 = 1;
+    this->unk_288 = 1;
+    func_80AC3350(this, globalCtx, 0.0f);
+}
+
+void func_80AC3A80(GlobalContext* globalCtx, EnOssan* this) {
+    this->unk_1FC = 3;
+    D_80AC8984[this->actor.params](globalCtx);
+    Interface_SetDoAction(globalCtx, 6);
+    this->unk_288 = 0;
+    this->unk_2C0 = 0;
+}
+
+void func_80AC3AE0(GlobalContext* globalCtx, EnOssan* this) {
+    func_80078884(NA_SE_SY_CURSOR);
+    this->unk_251 = 0;
+    this->unk_1FC = 8;
+}
+
+void func_80AC3B18(EnOssan* this, GlobalContext* globalCtx, Player* player) {
+    this->unk_1F0 = this->actor.yawTowardsLink - this->actor.shape.rot.y;
+    if (func_8002F194(&this->actor, globalCtx)) {
+        // Start talking
+        osSyncPrintf(VT_FGCOL(YELLOW) "★★★ 会話開始！！ ★★★" VT_RST "\n");
+        player->stateFlags2 |= 0x20000000;
+        func_800BC590(globalCtx);
+        func_80AC3928(globalCtx, this, 0);
+    } else if (this->actor.xzDistFromLink < 100.0f) {
+        func_8002F2CC(&this->actor, globalCtx, 100);
+    }
+}
+
+#ifdef NON_MATCHING
+void func_80AC3BCC(GlobalContext* globalCtx, EnOssan* this) {
+    s8 cont1 = globalCtx->state.input[0].rel.in.x;
+    s8 cont2 = globalCtx->state.input[0].rel.in.y;
```

Comment:
these variable names suggest controller one and controller 2 are being used, but its the x and y axis on the control stick for controller 1. so id do cont1X and cont1Y or something